staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -58,11 +61,28 @@ func NamedCheck(name string, check func(r *http.Request) error) HealthzChecker {
 	return &healthzCheck{name, check}
 }
 
-// InstallHandler registers a handler for health checking on the path "/healthz" to mux.
+// InstallHandler registers handlers for health checking on the path
+// "/healthz" to mux. *All handlers* for mux must be specified in
+// exactly one call to InstallHandler. Calling InstallHandler more
+// than once for the same mux will result in a panic.
 func InstallHandler(mux mux, checks ...HealthzChecker) {
 	if len(checks) == 0 {
+		if glog.V(5) {
+			glog.Info("No default health checks specified. Installing the ping handler.")
+		}
 		checks = []HealthzChecker{PingHealthz}
 	}
+
+	if glog.V(5) {
+		// accumulate the names of checks for printing them out.
+		checkerNames := make([]string, 0, len(checks))
```

Comment:
do you mind to move that into a helper at the bottom of the file? Something like `checkerName(checks) []string`? Then you can also use `glog.V(5).Info` here and keep the func nice and clean.